### PR TITLE
Removed unnecessarily exposed ports

### LINKED ISSUE #48: Identify and un-expose ports of service that are not required over the internet

### PR DESCRIPTION
**Pull request by RyanWalker277**
Fixes #48

### CODE REVIEW

**Comment by RyanWalker277** [comment]
https://github.com/samagra-comms/docker-deploy/blob/c67da57c8830547baa362b11c2660056a1fed4f1/docker-compose.yml#L224-L229

I am a bit unsure about these ports, they dont seem to be used by any other service. Let me know if I can unexpose these as well.

File: docker-compose.yml [hunk 1]
```diff
@@ -269,8 +263,6 @@ services:
   graphql-formsdb:
     restart: always
     image: hasura/graphql-engine:latest
-    ports:
```
Comment (tushar5526) [inline docker-compose.yml:272]: This has to be exposed

**Comment by tushar5526** [comment]
Items to be exposed.

- everything having a UI viz hasura, aggregate-server etc.
- items not having a UI or used internally through docker compose network, Caches, DB, other services where internal ports are being used (eg: Kafka)

File: docker-compose.yml (hunk 1 above)
Comment (tushar5526) [reply docker-compose.yml:272]: similarly a lot of other things needs to be exposed.

**Review by tushar5526 — CHANGES_REQUESTED** [review]
You have not fixed every mount. Please go through all comments and resolve one by one.

File: docker-compose.yml [hunk 2]
```diff
@@ -376,8 +352,6 @@ services:
     volumes:
       - ./vault/vault.json:/vault/config/vault.json
       - vault-data:/vault/file
-    ports:
```
Comment (tushar5526) [inline docker-compose.yml:379]: Vault is needed publicly

**Pull request merged**